.github/workflows/ci-wordpress-rollback.yaml

```diff
@@ -1,7 +1,8 @@
-name: CI-Wordpress
+name: CI-Wordpress-Rollback
 
 # ----------------------------------------------------------------------------------------------------------
-# Pipeline rollback the release by one version on shared hosting by ansistrano ansible role
+# Pipeline to rollback the release by one version back.
+# In order to rollback release by one step back, you need to run this workflow.
```

Comment:
- pls explain here what you consider a "release". 
- how the "release" will be identified ? by tag by sha commit id?

Reply:
During the deployment, ansistrano looks at the current date on the runner, in fact, it performs `date -u +%Y%m%d%H%M%SZ`. Each new release is placed in a folder with a timestamp. 
We don't have to roll back exactly one release, in fact, this is the easiest way to return the previous version of the release.
If we know the release timestamp (i.e. the name of the release folder), then we can specify this the timestamp and ancistrano will make the release with this label current. I'll add more information to the action documentation.

Reply:
Added information to the action header

Reply:
can we adjust the naming of the folder from INT to YYY-.... like I mentioned previously in my other comments.

Reply:
already done